core/src/main/java/com/linecorp/armeria/common/util/EventLoopGroups.java

```diff
@@ -89,7 +89,9 @@ public static EventLoopGroup newEventLoopGroup(int numThreads, String threadName
 
         final TransportType type = TransportType.detectTransportType();
         final String prefix = threadNamePrefix + '-' + type.lowerCasedName();
-        return newEventLoopGroup(numThreads, new EventLoopThreadFactory(prefix, useDaemonThreads));
+        return newEventLoopGroup(numThreads, ThreadFactories.builderForEventLoops(prefix)
+                                                            .daemon(useDaemonThreads)
+                                                            .build());
```

Comment:
This builder is fluent for a new `ThreadFactory` with `TaskFunction` however I guess in most case we just need a pair of `(prefix, useDaemon)`.
How about adding a shortcut method for common cases.
I suggest to add the following methods:
```java
ThreadFactory newEventLoopThreadFactory(String prefix, boolean daemon);
ThreadFactory newThreadFactory(String prefix, boolean daemon);
```

Reply:
Added shortcut methods.
* `ThreadFactories.newEventLoopThreadFactory(String, boolean)`
* `ThreadFactories.newThreadFactory(String, boolean)`
*  Use shortcut method where applicable